database/Group4Database.py

```diff
@@ -110,10 +110,11 @@ def get_factions(self):
             query = "MATCH (f:{}) return f.factionId as factionId, f.name as name".format(NODE_FACTION)
             return session.run(query).data()
 
-    def get_persons_ranked(self, sentiment_type, session_id=None):
+    def get_persons_ranked(self, sentiment_type, session_id=None, reverse=None):
```

Comment:
man könnte den Parameter auch schon boolean machen und in der app.py als `request.args.get(QUERY_PARAM_EXCLUDE_REVERSE) == 'true'` übergeben